src/components/SeachInput.svelte

```diff
@@ -1,5 +1,12 @@
 <script lang="ts">
+  import { onMount } from "svelte";
+
   export let value = "";
+  export let loaded = false;
```

Comment:
Should we name this prop `disabled`?

Because this prop will be used to disabled directly.

Reply:
The intent is to disable when the client-side code has not yet been loaded.

Otherwise, the reader of this code may wonder why we have `disabled = true` which is later changed on mount.

Reply:
I will rename it to "componentIsHydrated" instead.

Reply:
Understood, please change to `componentIsHydrated `.